packages/raystack/components/meter/meter.module.css

```
@@ -53,7 +55,10 @@
 
 .circularSvg {
   --rs-meter-track-size: var(--rs-space-2);
-  --rs-meter-radius: calc((var(--rs-space-14) - var(--rs-meter-track-size) * 2) / 2);
+  --rs-meter-radius: calc(
+    (var(--rs-space-14) - var(--rs-meter-track-size) * 2) /
```

Stylelint (17.14.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)